app/utils/rw_api.py

```diff
@@ -185,3 +195,210 @@ async def signup(name: str, email: str) -> User:
         )
 
     return User(**response.json()["data"])
+
+
+async def create_rw_geostore(
+    payload: RWGeostoreIn, x_api_key: str | None = None
+) -> RWGeostoreResponse:
+    url = f"{RW_API_URL}/v1/geostore"
+
+    async with AsyncClient() as client:
+        if x_api_key is not None:
+            response: HTTPXResponse = await client.post(
+                url, json=payload.dict(), headers={"x-api-key": x_api_key}
+            )
+        else:
+            response = await client.post(url, json=payload.dict())
+
+    if response.status_code == 201:
+        return RWGeostoreResponse.parse_obj(response.json())
+    else:
+        raise HTTPException(response.status_code, response.text)
+
+
+async def proxy_get_geostore(
+    geostore_id: str, x_api_key: str | None = None
+) -> RWGeostoreResponse:
+    url = f"{RW_API_URL}/v2/geostore/{geostore_id}"
+
+    async with AsyncClient() as client:
+        if x_api_key is not None:
+            response: HTTPXResponse = await client.get(
+                url, headers={"x-api-key": x_api_key}
+            )
+        else:
+            response = await client.get(url)
+
+    if response.status_code == 200:
+        return RWGeostoreResponse.parse_obj(response.json())
+    else:
+        raise HTTPException(response.status_code, response.text)
+
+
+async def calc_area(
+    payload: Dict, x_api_key: str | None = None
+) -> RWCalcAreaForGeostoreResponse:
+    url = f"{RW_API_URL}/v1/geostore/area"
```

Comment:
This should be `v2`, right?
```suggestion
    url = f"{RW_API_URL}/v2/geostore/area"
```

Reply:
> This should be `v2`, right?

Same as the create geostore endpoint (v1 and v2 both work).

Reply:
Resolved in previous comment: https://github.com/wri/gfw-data-api/pull/621#discussion_r1925501964